diagnostic.md

```diff
@@ -45,18 +54,22 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+The 1st 'weather' expression sets 'weather' equal to the value associated with "sunny"
+The 2nd 'weather' experssion sets the absolute value of 'weather' to 'sunny' as a string
+
+
 
 ## Question 5
 
 ```js
 //We'll learn about require later in the course
 const ask = require('../lib/ask.js');
 
-let answer = 'not empty';
+let answer = '1';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer === '1') {
    console.log("You guessed it!");
    break;
   }
```

Comment:
You're halfway there! The question also asked you to console.log a message to the user prompting them to enter a number between 1 and 10. Take a look at this solution, and let us know if you have questions.
```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```